Allow usage of global configuration values for `TryExamples` directive if provided by user (#161)
jupyterlite/jupyterlite-sphinx: docs/directives/try_examples.md

```

The `try_examples` directive has these options:

* `:height:` To set a specific value for the height of the [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) containing the embedded notebook.
* `:button_text` To customize the text of the button that replaces the rendered examples with an embedded notebook. The default value is `"Try it with JupyterLite!"`, or the value of the global configuration variable `try_examples_global_button_text` if it is set in `conf.py`.
* `:theme:` This works the same as for the other JupyterLite-Sphinx directives.
* `:example_class:` An html class to attach to the outer container for the rendered
examples content and embedded notebook. This can be used in a custom css file to allow
for more precise customization, eg. different button styles across different examples.
The default value is the value of the global configuration variable `try_examples_global_theme`, if it is set in `conf.py`.

> * `:warning_text:` Prepend a markdown cell to the notebook containing this text, styled to make it clear this is intended as a warning. The default value is the value of the global configuration variable `try_examples_global_warning_text` if it is set in `conf.py`, otherwise no warning text is added.

Reviewer: I think it makes more sense for you not to change these, and then mention that `try_examples_global` config options apply both to automatically inserted `try_examples` directives, and manually inserted ones when the global configuration is discussed. For instance, I wouldn't consider the value of `try_examples_global_warning_text` to be a default, I think of it as setting to a non-default value in the global config. Also, I think it's better not to refer so heavily to the `try_examples_global` config options before they are introduced.

Author: I agree with all of the three points. I have included these details under the "Global Configuration" section and reverted two of my previous commits in 7dad6e20120067617c2213eaa34515943038e096, and these details do make more sense there.

The button text, theme, and warning text can be set globally with the config variables

> `try_examples_global_button_text`, `try_examples_global_theme`, and `try_examples_global_warning_text` in `conf.py`;
> and these apply both to automatically inserted directives and to manually inserted ones. The
> priority of these global settings is lower than that of the specific options set for a manually
> inserted directive, which will override the global settings on a per-directive basis.

Reviewer: ```suggestion
`try_examples_global_button_text`, `try_examples_global_theme`, and `try_examples_global_warning_text` in `conf.py`;
these apply both to automatically and manually inserted directives. Options set explicitly in a directive will
override the global configuration.
```

A semicolon should link two independent clauses which form a logical whole. The effect is to imply a pause of shorter duration than a hard stop. Starting the second clause with an "and" makes it a dependent clause. Also I think this can be expressed more concisely.

try_examples_global_warning_text = "Interactive examples are experimental and may not always work as expected."
```

> There is no option to set a global specific height because the proper height

Reviewer: I'm not sure this is an improvement. It's distance from a contrasting statement makes the "However" seem out of place to me.

Author: Should we allow overriding the height – we don't do that currently?

Author: I got confused with something else, please don't mind. Since we do have a method to set a global minimum height with the JSON file, I think removing "However" is okay – it makes it feel like there is a setback when there is none.

Author: Addressed this in 998cd260a78a8b337682116d8a87404b0f19cd0a
